src/Stratis.Bitcoin/Configuration/Logging/LoggingConfiguration.cs

```diff
@@ -19,11 +20,24 @@ namespace Stratis.Bitcoin.Configuration.Logging
     /// </summary>
     public class ExtendedLoggerFactory : LoggerFactory
     {
+        private const string NLogConfigFileName = "NLog.config";
+
         /// <summary>Configuration of console logger.</summary>
         public ConsoleLoggerSettings ConsoleSettings { get; set; }
 
         /// <summary>Provider of console logger.</summary>
         public ConsoleLoggerProvider ConsoleLoggerProvider { get; set; }
+
+        /// <summary>Loads an NLog.config file from the <see cref="DataFolder"/>, if it exists</summary>
```

Comment:
missing `.`

also i'd replace `NLog.config` with either `configuration` or `<see cref="NLogConfigFileName"/>`

Reply:
`configuration` is not specific, and <see cref="NLogConfigFileName"/> is obscure. NLog.config is exactly the name of the file that it loads.

Reply:
I can inline the constant if you'd prefer?

Reply:
i'd probably prefer `logging configuration`, inlining constants is not a good idea